src/app/containers/MediaPlayer/helpers/metadata/index.js

```diff
@@ -4,7 +4,15 @@ import pathOr from 'ramda/src/pathOr';
 const DEFAULT_IMAGE_RES = '1024x576';
 
 export const getThumbnailUri = aresMetadataBlock => {
-  const imageUrl = pathOr('', ['model', 'imageUrl'], aresMetadataBlock);
+  let imageUrl = pathOr('', ['model', 'imageUrl'], aresMetadataBlock);
+
+  if (imageUrl.includes('http')) {
```

Comment:
```suggestion
  if (imageUrl.startsWith('http')) {
```

Reply:
Considered that - didn't know about `startsWith`, nice! 👍